book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(person, date)
+    rental = Rental.new(date, self, person)
+    @rentals << rental
+    person.add_rental(rental)
+  end
```

Comment:
- [ ] You have a lot of unnecessary code here. All we need is the first instantiation on line 11. The rest are repetitive of the first line. Creating a rental on line 11, already adds it to the rentals array in the Rental class, so there's no need for line 12. There is also no need to call `add_rental` in line 13 (you are not calling it correctly either), the methods should be independent of each other.

  ```
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
  ```